J/JpegTurbo/build_tarballs.jl

```diff
@@ -1,14 +1,14 @@
 using BinaryBuilder
 
 name = "JpegTurbo"
-upstream_version = v"3.1.0"
-version = v"3.1.1" # Needed to change version number to bump compat bounds, next time can go back to follow upstream
+upstream_version = v"3.1.1"
+version = v"3.1.2" # Needed to change version number to bump compat bounds, next time can go back to follow upstream
```

Comment:
Can't we go back to following upstream as suggested by the comment?

Reply:
Are you suggesting wrapping both JpegTurbo 3.1.0 and 3.1.1 as `JpegTurbo_jll` 3.1.1?

Reply:
So you are suggesting not doing an actual version bump here and just get a new build number? I thought we didn't want to do that generally.

Reply:
I don't like it either but I don't have a strong opinion. Omitting the version bump means that any problems with the new version (incompatibility, or caused by problems with the build systems) cannot be easily addressed downstream. Instead, we'd have to yank the new version from the registry.